docs/source/getting-started/registering-first-model/step1-register-model.rst

```diff
@@ -0,0 +1,75 @@
+Register a Model
+=================
+
+Throughout this tutorial we will leverage a local tracking server and model registry for simplicity.
+However, for production use cases we recommend using a 
+`remote tracking server <https://mlflow.org/docs/latest/tracking/tutorials/remote-server.html>`_.
+
+Step 0: Install Dependencies
+----------------------------
+.. code-section::
+    .. code-block:: bash
+
+        pip install --upgrade mlflow
+
+Step 1: Register a Model
+--------------------------------
+
+MLflow has lots of model flavors. In the below example, we'll leverage scikit-learn's 
+RandomForestRegressor to demonstrate the simplest way to register a model, but note that you
+can leverage any `supported model flavor <https://mlflow.org/docs/latest/models.html#built-in-model-flavors>`_.
+
+.. code-section::
+    .. code-block:: python 
+        :name: create-model 
+
+        from sklearn.datasets import make_regression
+        from sklearn.ensemble import RandomForestRegressor
+        from sklearn.metrics import mean_squared_error
+        from sklearn.model_selection import train_test_split
+
+        import mlflow
+        import mlflow.sklearn
+
+        with mlflow.start_run() as run:
+            X, y = make_regression(n_features=4, n_informative=2, random_state=0, shuffle=False)
+            X_train, X_test, y_train, y_test = train_test_split(
+                X, y, test_size=0.2, random_state=42
+            )
+
+            params = {"max_depth": 2, "random_state": 42}
+            model = RandomForestRegressor(**params)
+            model.fit(X_train, y_train)
+
+            # Log parameters and metrics using the MLflow APIs
+            mlflow.log_params(params)
+            
+            y_pred = model.predict(X_test)
+            mlflow.log_metrics({"mse": mean_squared_error(y_test, y_pred)})
+
+            # Log the sklearn model and register as version 1
+            mlflow.sklearn.log_model(
+                sk_model=model,
+                artifact_path="sklearn-model",
+                input_example=X_train, 
+                registered_model_name="sk-learn-random-forest-reg-model", 
+            )
+
+
+.. code-block:: bash
+    :caption: Example Output
+
+    Successfully registered model 'sk-learn-random-forest-reg-model'.
+    Created version '1' of model 'sk-learn-random-forest-reg-model'.
+
+Great! We've registered a model. 
```

Comment:
I think it would be helpful to explain the concept of registered model and model versions. Since we're assuming the user is completely new to the model registry and our code snippet creates both a registered model and its first model version.